website/docs/cluster-management/managing-clusters-without-capi.mdx

```diff
@@ -182,18 +178,80 @@ kubectl create secret generic demo-01-kubeconfig \
 </TabItem>
 </Tabs>
 
-### Connect a cluster
+## Add a Cluster Bootstrap Config
 
-:::tip Before you start!
+This step ensures that Flux gets installed into your cluster. Create a cluster bootstrap config as follows:
 
-Make sure you've
+```bash
+ kubectl create secret generic my-pat --from-literal GITHUB_TOKEN=$GITHUB_TOKEN
+```
+
+import CapiGitopsCDC from "!!raw-loader!./assets/bootstrap/capi-gitops-cluster-bootstrap-config.yaml";
+
+Download the config with:
+
+<BrowserOnly>
```

Comment:
Not sure why but this content is not rendered at all see https://staging.docs.gitops.weave.works/docs-changes-june7/docs/next/cluster-management/managing-clusters-without-capi/#add-a-cluster-bootstrap-config

Reply:
I don't get it either